frontend/src/types.ts

```diff
@@ -159,6 +159,8 @@ export type OdhApplication = {
     betaText?: string | null;
     shownOnEnabledPage: boolean | null;
     isEnabled: boolean | null;
+    isInstalled: boolean | null;
+    canInstall: boolean | null;
```

Comment:
Fairly certain these are not required -- we don't ever attach this information to the OdhApplication afaik.

Reply:
Updated.